index.js

```diff
@@ -10,7 +10,7 @@ class APSInstance extends InstanceBase {
 		super(internal)
 	}
 
-	updateConfig(config) {
+	async updateConfig(config) {
```

Comment:
this would be `async configUpdated` see [here](https://github.com/bitfocus/companion-module-base/wiki/Upgrading-a-module-built-for-Companion-2.x#12-updating-your-main-class) below where it talks about `init`.

Reply:
Sorry, it is updated to `async configUpdated` in both modules now